client/lib/domains/constants.js

```diff
@@ -17,7 +17,40 @@ const registrar = {
 	MAINTENANCE: 'Registrar TLD Maintenance'
 };
 
+const tlds = {
```

Comment:
@yurynix Good idea. What do you think about a `tlds` array for each object in `products-list` that has `is_domain_registration: true`?